spec/services/usps_in_person_proofing/transliterator_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe UspsInPersonProofing::Transliterator do
+  describe '#transliterate' do
+    let(:sut) { UspsInPersonProofing::Transliterator.new }
```

Comment:
what does `sut` mean?

Reply:
`sut` is short for `subject under test` or `system under test`.

Reply:
that's not really a common acronym we use in our codebase, and RSpec has a specific notion of a subject, so I'd consider using a named subject:
```suggestion
    subject(:transliterator) { UspsInPersonProofing::Transliterator.new }
```
and then we can refer to it by either `transliterator` or `subject` in any of the specs

Reply:
Added.